VAMobile/src/components/AnnouncementBanner.tsx

```diff
@@ -0,0 +1,68 @@
+import React, { FC } from 'react'
+import { Linking, Platform, Pressable, ViewStyle } from 'react-native'
+
+import { BackgroundVariant, Box, BoxProps, TextView, VAIcon } from 'components'
+import { useTheme } from 'utils/hooks'
+
+import colors from '../styles/themes/VAColors'
+
+interface AnnouncementBannerProps {
+  /** Text for announcement title */
+  title: string
+  /** link to navigate to, excluding the prefix */
+  link: string
+}
+
+const AnnouncementBanner: FC<AnnouncementBannerProps> = ({ title, link }: AnnouncementBannerProps) => {
+  const theme = useTheme()
+
+  const boxProps: BoxProps = {
+    py: theme.dimensions.formMarginBetween,
+    px: theme.dimensions.cardPadding,
+    backgroundColor: theme.colors.buttonBackground.announcementBanner as BackgroundVariant,
+    style: {
+      shadowColor: colors.black,
+      ...Platform.select({
+        ios: {
+          shadowOffset: { width: 0, height: 6 },
+          shadowOpacity: 0.2,
```

Comment:
```suggestion
          shadowOpacity: 0.4,
```
Based on the design in Figma, the opacity should be 40%. I'd double-check how this looks in the app after updating though, because to me, this already looks pretty close to the design

Reply:
40% opacity makes it look a bit darker than how it looks on Figma, but I'll changed it to 40% to match the design

Reply:
Yeah, it could be my eyes, but I also noticed the drop shadow on Android looks a bit darker than the Figma design. Might have to lower that one as well. We can see what design says in visual QA